server/startLetsEncrypt.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Configures and starts express server via Let's Encrypt.
+ *
+ * Events are fired during initialisation to allow customisation, including:
+ *   - onHttpServerCreated
+ *
+ * consumed by lib/core/start.js
+ *
+ * @api private
+ */
+
+var http = require('http');
+var https;
+try {
+	// Use spdy if available
+	https = require('spdy');
+} catch (e) {
+	https = require('https');
+}
+var path = require('path');
+var LEX = require('letsencrypt-express');
+var lexDir = path.join(require('os').homedir(), 'letsencrypt', 'etc');
```

Comment:
Path is more consistent with our naming conventions, let's go with `letsencryptPath`